specs/core/0_beacon-chain.md

```diff
@@ -998,6 +1011,10 @@ def integer_squareroot(n: int) -> int:
     return x
 ```
 
+#### `BLSVerify`
+
+`BLSVerify` is a function for verifying a BLS12-381 signature, defined in the [BLS12-381 spec](https://github.com/ethereum/eth2.0-specs/blob/master/specs/bls_verify.md).
```

Comment:
Ah, I thought `BLS Verification` spec is "Boneh-Lynn-Shacham signatures verification" because it's not talking about the details of the curve, no?

Reply:
Well that doc is doing a BLS signature verification for a specific BLS12-381 curve

Reply:
renamed spec ref to "BLS Verification spec"